common/presenters/date-table/cell-renderers.js

```diff
@@ -0,0 +1,76 @@
+const { format } = require('date-fns')
+const { isNil } = require('lodash')
+
+const i18n = require('../../../config/i18n')
+
+const freeSpacesCellData = {
```

Comment:
So the idea is that each population cell has a potential link, and some text based upon the data and some `i18n`. These two functions provide that, and are assembled using the `genericValueCellContent` function.

This also allows all permutations of the `url` and `content` to be tested, without needing to build the full table to do so.